static/src/partials/wrapper.html

```diff
@@ -11,8 +11,13 @@
   <meta charset="utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="description" content="Search, discover, visualize, refine, and access NASA Earth Observation data in your browser with Earthdata Search" data-react-helmet="true" />
+  <meta name="description"
+    content="Search, discover, visualize, refine, and access NASA Earth Observation data in your browser with Earthdata Search"
+    data-react-helmet="true" />
   <link rel="shortcut icon" href="/favicon.ico?v=1.1" />
+  <script async type="text/javascript"
+    src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=NASA&subagency=GSFC"
+    id="_fed_an_ua_tag"></script>
```

Comment:
Closing `</script>` tag should be on a newline since it has a multi-line body.

Reply:
This script tag should go in after the body if possible. Otherwise it could cause some blocking. It probably makes more sense to throw it in the `body.html` partial near the feedback app. Seems like this is something we might only want to run when not in the `development` env too.